packages/fiber/src/core/store.ts

```diff
@@ -135,10 +142,15 @@ export type RootState = {
   setEvents: (events: Partial<EventManager<any>>) => void
   /**
    * Shortcut to manual sizing
-   *
-   * @todo before releasing next major version (v9), re-order arguments here to width, height, top, left, updateStyle
    */
-  setSize: (width: number, height: number, updateStyle?: boolean, top?: number, left?: number) => void
+  setSize: (
+    width: number,
+    height: number,
+    /** @deprecated `updateStyle` is now disabled for OffscreenCanvas and will be removed in v9. */
+    updateStyle?: boolean,
+    top?: number,
+    left?: number,
+  ) => void
```

Comment:
We should consider an overload for this for the `setSize(width, height, top, left)` signature by checking if `updateStyle` is a number and if `top` is defined but not `left`.